### PR TITLE
Fix licences

### PR DESCRIPTION
**Pull request by seionmoya**
Since only complaining does not resolve the situation from #330, here's my proposal to fix the issues discussed in addition to other cleanup work:

> Remove outdated readmes

Since DE/FR/PO/KO are in english, I decided to remove these from the repository to reduce clutter.

> Unified readmes

Now all readmes will display both the ko-fi links and repositories as `README_CN.md` did.

> - Remove mention of code being MIT in the readme
> - Remove mention of code being unlicenced in the readme

Since `LICENSE.md` states the project to be NCSA, the project henceforth will no longer be tripple-licenced. The project in the previous commits will retain the original licencing model and all rights within those commits are retained.

Starting from this PR, all code within the repository will solely be covered under NCSA. This is possible because all previous code has been covered under NCSA alongside the other licenses (MIT, unlicence).

> - Include proper attribution

An author was previously mistakenly removed from the acknowledgement list and has been restored.

> - Removed "Merijn Hendriks" from attribution and changed SPT-AKI to SPT

This is to reflect SPT's updated license, visible here:
https://dev.sp-tarkov.com/SPT/Modules/src/branch/master/LICENSE.md

> - Removed paulov's ko-fi link in readme

As requested by paulov, I think it's important to honor his wishes for this.

### CODE REVIEW

**Comment by SlejmUr** [comment]
As I have an original copy of SIT.Core I tried to get all my contributors from that since most of codes are helped with the project.

Can access it here:
https://github.com/SlejmUr/SIT.Core/commits/master/

Contributors (Github names) to a to-be : "CONTRIBUTORS.txt"
ae_King
ArchangelWTF
belettee
ChewieDM
cocorico8
debisuchan
derbismarck
devbence
donunai2333
evileyecc1
ezra-sg
jeremymouton
JimWails
JonBons
kapdon
konstantin90s
Lacyway
landonk89
langhillie
Luckz
lukehemmin
mihaicm93
mikeyybroo
mpaulustsx
OkaMoez
paulov-t
Plootie
ppyLEK
privateryann1
Sayoki-Yukina
shibdib
SlejmUr
Starkrights
TehFl0w
trippyone
w123456w30w
yhl452493373

(alphabetical order)

**Review by ArchangelWTF — APPROVED** [review]
lgtm

**Review by SlejmUr — APPROVED** [review]

**Review by kahvana166 — APPROVED** [review]

**Review by samurailink3 — APPROVED** [review]

**Review by EFHDev — APPROVED** [review]
It overall looks good, and proper, good shit,

File: README.md [hunk 1]
```diff
@@ -103,14 +99,15 @@ Our Discord invite link can be found at https://stayintarkov.com/discord
...
+* [Mihai](https://github.com/mihaicm93) | [Ko-Fi Donations](https://ko-fi.com/mmihai)
 * [Trippy](https://github.com/trippyone)
-* [Bullet](https://github.com/devbence)
+* [Lacyway](https://github.com/lacyway)
+* [Bullet](https://github.com/devbence) | [Ko-Fi Donations](https://ko-fi.com/bullet4prz)
 * [Dounai](https://github.com/dounai2333)
 * [SPT-Aki team](https://www.sp-tarkov.com/) (Credits provided on each code file used and much love to their Dev team for their support)
 * [DrakiaXYZ](https://github.com/DrakiaXYZ/)
-* [Contributors](https://github.com/stayintarkov/StayInTarkov.Client/graphs/contributors) and the original contributors of Paulov's SIT.Core
+* [Contributors](https://github.com/stayintarkov/StayInTarkov.Client/graphs/contributors) | [SIT.Core](https://github.com/seionmoya/StayInTarkov.Client/blob/fix-licences/CONTRIBUTORS.md)
```
Comment (SlejmUr) [inline README.md:110]: [SIT.Core](https://github.com/stayintarkov/StayInTarkov.Client/blob/master/CONTRIBUTORS.md)
 
 *for to be merged

**Comment by EFHDev** [comment]
Seeing as this isnt going to go anywhere due to the leadership, i highly suggest any level headed SIT contributors to not touch any of the code as it is not owned by SIT in its current state. 

You cannot relicense this as NCSA, or anything else without proper attribution for not only lacy's work, but everyone's who worked on SIT. Its just a line of text and yet, you refuse to do what is deserved of your contributors, 

I hoped the new management would be able to look past paulov wanting to turn this community into a compaction, that we would finally be able to our friends at SIT back, like we used to back in the SIT server days. The EmuTarkov community should be working together. Not against each other. None of us would exist without the ones who came before us. 

I also thought without paulov you would drop your censorship policy, but looking into #webhook in the discord you see no mention of my PR, and then after mine was deleted, you then deleted paulovs. 
image
image

I give up trying to help you guys. And i feel like a lot of other people feel the same. 
Good luck in your future endeavors, Mihai, bullet, and whatever other leadership left in SIT.

**Review by mihaicm93 — APPROVED** [review]

**Pull request merged**